BaseClasses.py

```diff
@@ -400,7 +455,17 @@ def get_game_worlds(self, game_name: str):
                      player not in self.groups and self.game[player] == game_name)
 
     def get_name_string_for_object(self, obj: HasNameAndPlayer) -> str:
-        return obj.name if self.players == 1 else f'{obj.name} ({self.get_player_name(obj.player)})'
+        if self.players == 1:
```

Comment:
Initially, I modified the name set in `MultiWorld.player_name` in `MultiWorld.add_group()`, when creating a `Group` for item links, to include all the players in the group. This seemed to work, but I suspect was not safe to do.